packages/grid_client/src/high_level/twinDeploymentHandler.ts

```diff
@@ -27,13 +29,14 @@ class TwinDeploymentHandler {
     if (id) {
       const c = await this.tfclient.contracts.get({ id });
       if (c && c.twinId !== this.config.twinId) {
-        throw Error(`Name contract with name ${name} is already reserved`);
+        throw new ValidationError(`Name contract with name ${name} is already reserved`);
       }
     }
     try {
       return await this.tfclient.contracts.createName({ name });
     } catch (e) {
-      throw Error(`Failed to create name contract ${name} due to ${e}`);
+      //TODO ERROR should be handled in tfchain
```

Comment:
remove the comment, or is it necessary as a future reminder?

Reply:
yes those TODO need to be implemented in the future. the mentioned client